packages/connect-react-query/src/index.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021-2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export type {
+  ConnectQueryKey,
+  ConnectPartialQueryKey,
+} from "./connect-query-key.js";
+export { disableQuery } from "./utils.js";
+export { useTransport, TransportProvider } from "./use-transport.js";
+export type {
+  UseInfiniteQueryOptions,
+  createUseInfiniteQueryOptions,
```

Comment:
Yes, these are needed when used with `useQueries({ queries: [ createUseQueryOptions(...) ] })` in order to define parallel queries. I thought perhaps we could define our own `useQueries()` but that would mean it would be impossible to mix connect queries with other external queries.